<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader_message</class>
 <widget class="QWidget" name="reader_message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>读者信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 20pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>身份证号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="reader_name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="age">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_num">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_num">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>630</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>630</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
